packages/docs/operations-manual/key-management.md

```diff
@@ -4,65 +4,209 @@
+
+# Authorize the vote signing key. Note that the Locked Gold Account private key must be available.
+celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role vote --signer $KEY_TO_AUTHORIZE --signature $PROOF_OF_POSSESSION --useLedger
+
+# Confirm that the vote signer was authorized
+celocli account:show $LOCKED_GOLD_ACCOUNT
+
+# You can also look up account info via the authorized signer
+celocli account:show $KEY_TO_AUTHORIZE
+```
+
+## Authorized Validator Signers
+
+Any Locked Gold Account key may optionally authorize an undesignated key as a validator signer. Authorized validator signing keys can be used to register and manage a validator or validator group on behalf of the Locked Gold Account. If the authorized validator key is used to register a validator it is also used to sign consensus messages.
+
+### Authorized Validator Signers for Validator Groups
+An authorized validator signer key that will be used to register a validator group can be used to send group management transactions (e.g. register, add member A, queue commission update to 0.25, etc.) Because this key does not participate directly in consensus it can be stored directly on a Ledger hardware wallet.
+
+### Authorized Validator Signers for Validators
+An authorized validator signer key that will be used to register a validator can be used to send validator management transactions (e.g. register, affiliate with group A, etc.) This key will also be used to sign consensus messages and thus *cannot be stored directly on a Ledger hardware wallet* as signing consensus messages is not currently supported by the Celo Ledger App.
 
-The Account key is the most sensitive key because it can authorize the other keys and access locked gold. As it should be used quite infrequently, it is highly recommended for important Account keys (i.e. Validators or accounts with high balances) to remain as secure as possible. We recommend them to be offline, ideally in cold storage, or on a hardware wallet.
+Note that the validator signing key must first be used to generate a "proof-of-possession" indicating that key's willingness to be authorized as a signer on behalf of the Locked Gold Account.
 
-Signing keys are less sensitive, and more frequently used, than the Account key, but should still be safe-guarded. Validator signing keys, in particular, could be used to execute a double-signing attack if compromised, which may result in [slashing](../celo-codebase/protocol/proof-of-stake/penalties).
+
+### Authorizing a Validator Signer
+An undesignated key may be authorized as a validator signer on behalf of a Locked Gold Account by runnign the following commands:
+
+```shell
+# Create a proof-of-possession. Note that the signer private key must be available.
+# Note that the signing key should not be kept on a Ledger if it will be used to run a Validator
+celocli account:proof-of-possession --account $LOCKED_GOLD_ACCOUNT --signer $KEY_TO_AUTHORIZE --useLedger
```

Comment:
Isn't this `useLedger` in contradiction with the comment above it?

Reply:
Not strictly, IMO, since you can do this for a validator group as well. But switched it around to be more clear, hopefully